cmd/internal/find/primary/meta/stringPredicate.go

```diff
@@ -0,0 +1,29 @@
+package meta
+
+import (
+	"fmt"
+
+	"github.com/puppetlabs/wash/cmd/internal/find/primary/errz"
+)
+
+// StringPredicate => [^-]+
```

Comment:
I think the pattern needs something more, `[^-]+` is incomplete unless you mean to say the string can't have a dash anywhere in it. That doesn't seem to be enforced by the predicate.

Reply:
You're right, that was a brainfart on my part. Should be `[^-].*`. Will update later.